plone/app/registry/browser/records.py

```diff
@@ -23,6 +24,12 @@ def _starts_with(s, v):
 
 class RecordsControlPanel(BrowserView):
 
+    @property
+    def control_panel_url(self):
+        return u"{0}/@@overview-controlpanel".format(
+            getSite().absolute_url()
```

Comment:
Here, not context is used, but getSite - contrary to the other change, where context is used.
The reason is, that the context of the ``@@configuration_registry`` controlpanel is the registry itself, where we cannot call the ``@@overview-controlpanel``. Also, it's not given, that the registries parent is a portal. But most likely, when this control panel is called on a registry of a subsite, the subsite is also configured to allow the overview controlpanel to be shown. If not in a subsite, it's the portal root.